aviary/subsystems/propulsion/propeller_performance.py

```diff
@@ -276,12 +241,76 @@ def setup(self):
                 Aircraft.Engine.PROPELLER_DIAMETER,
                 "density_ratio",
                 Dynamic.Mission.INSTALLATION_LOSS_FACTOR,
-                "adv_ratio",
+                "advance_ratio",
                 "power_coefficient",
             ],
             promotes_outputs=[
                 "thrust_coefficient_comp_loss",
-                "prop_thrust",
+                "propeller_thrust",
                 "propeller_efficiency",
                 "install_efficiency",
             ])
+
+
+class UncorrectShaftPower(om.Group):
```

Comment:
I believe this uncorrection will be useful for shaft RPM and thrust, so it might be better to have a more generalized one that can be used.

Reply:
Good idea, I can move it to propulsion's utils.py and have it uncorrect any data you feed it